src/containers/LoginFormContainer.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from "react";
+import FacebookLogin from "react-facebook-login";
+import GoogleLogin from "react-google-login";
+import { BASE_URL, API_FB_ID, API_GOOGLE_ID } from "../constants";
+import axios from "axios";
+
+export default class LoginFormContainer extends Component {
+  state = {
+    isLoggedIn: false,
+    userID: "",
+    firstName: "",
+    lastName: "",
+    email: "",
+    picture: "",
+    token: ""
+  };
+
+  componentClicked = () => console.log("clicked");
+
+  responseFacebook = response => {
+    console.log(response);
+
+    this.setState({
```

Comment:
I suggest making local copy of state, modyfing it, then passing whole modified object to setState, like:
```suggestion
    let state = this.state;
    //some magic with local state
    this.setState(state);
```
This way we know explicitly what is being passed to setState and if needed, do some prior validation on state object. Same goes for every usage of setState in PR